alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/action/IndexMonitorResponseTests.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.alerting.action
+
+import com.amazon.opendistroforelasticsearch.alerting.core.model.CronSchedule
+import com.amazon.opendistroforelasticsearch.alerting.model.Monitor
+import org.elasticsearch.common.io.stream.BytesStreamOutput
+import org.elasticsearch.common.io.stream.StreamInput
+import org.elasticsearch.rest.RestStatus
+import org.elasticsearch.test.ESTestCase
+import java.time.Instant
+import java.time.ZoneId
+
+class IndexMonitorResponseTests : ESTestCase() {
+
+    fun `test index monitor response with monitor`() {
+        val cronExpression = "31 * * * *" // Run at minute 31.
+        val testInstance = Instant.ofEpochSecond(1538164858L)
+
+        val cronSchedule = CronSchedule(cronExpression, ZoneId.of("Asia/Kolkata"), testInstance)
+        val req = IndexMonitorResponse("1234", 1L, 2L, 0L, RestStatus.OK,
+                Monitor(
+                "123",
+                        0L, "test-monitor", true, cronSchedule, Instant.now(),
+                        Instant.now(), 0, mutableListOf(), mutableListOf(), mutableMapOf()
+
+                ))
```

Comment:
Please fix formatting so it is consistent for the input attributes

Reply:
Fixed formatting.